iceoryx_examples/iceperf/roudi_main_static_config.cpp

```diff
@@ -25,7 +25,15 @@ int main(int argc, char* argv[])
     static constexpr uint32_t ONE_MEGABYTE = 1024U * 1024;
 
     iox::config::CmdLineParserConfigFileOption cmdLineParser;
-    cmdLineParser.parse(argc, argv);
+    auto cmdLineArgs = cmdLineParser.parse(argc, argv);
+    if (cmdLineArgs.has_error())
```

Comment:
Should we have a default case ?  What information does the user get back when the error is not `UNKNOWN_OPTION_USED` ?

Reply:
This is a topic for a future pull request. `m_run` in `CmdLineArgs_t` could be replaced with `CmdLineParserResult::INFO_OUTPUT_ONLY`. Currently no other error that `UNKNOWN_OPTION_USED` is used. So it's fine.

Reply:
Currently yes.. but what happens when someone adds another error and forgets to update this? 
All good if covered later though.

Reply:
Created #539